.github/workflows/push-changes.yaml

```diff
@@ -0,0 +1,25 @@
+name: Push Changes
+
+on:
+  push:
+    branches: [main]
+  workflow_run:
+    workflows:
+      - Cypress
+      - Docker
+    types:
+      - completed
+
+jobs:
+  dispatch:
+    strategy:
+      matrix:
+        repo: ['devfile/registry-support']
+    runs-on: ubuntu-latest
+    steps:
+      - name: Repository dispatch
+        uses: peter-evans/repository-dispatch@v1
+        with:
+          token: ${{ secrets.REPO_ACCESS_TOKEN }}
+          repository: ${{ matrix.repo }}
+          event-type: build
```

Comment:
Could you do me a favour and add the following below `event-type`?

```
client-payload: '{"repo": "${{ github.event.repository.name }}", "sha": "${{ github.sha }}"}'
```

This allows us to include the repo name and commit sha in the dummy commit message that triggers the redeploy.

Reply:
I can add this later too, but figured this would save me some time 🙂

Reply:
could probably do this for all the dispatches used in devfile repos, to keep it uniform